packages/instantsearch.js/src/connectors/frequently-bought-together/connectFrequentlyBoughtTogether.ts

```diff
@@ -123,12 +124,36 @@ const connectFrequentlyBoughtTogether: FrequentlyBoughtTogetherConnector =
           };
         },
 
-        getWidgetRenderState({ results }) {
+        getWidgetRenderState({ results, helper, instantSearchInstance }) {
+          if (!sendEvent) {
+            sendEvent = createSendEventForHits({
+              instantSearchInstance,
+              getIndex: () => helper.getIndex(),
+              widgetType: this.$$type,
+            });
+          }
+
           if (results === null || results === undefined) {
-            return { hits: [], widgetParams };
+            return { hits: [], sendEvent, widgetParams };
```

Comment:
Hey Dhaya, as far as I know, queryID are only available on search query (not recommend queries).
I think the queryID mentioned here does not relate to recommend.